WordPress/Classes/Models/BlogSettings.swift

```diff
@@ -0,0 +1,203 @@
+import Foundation
+
+
+/**
+ *  @class           BlogSettings
+ *  @brief           This class encapsulates all of the settings available for a Blog entity
+ */
+public class BlogSettings : NSManagedObject
+{
+    // MARK: - Relationships
+    
+    /**
+    *  @details Maps to the related Blog.
+    */
+    @NSManaged var blog : Blog?
+    
+
+    
+    // MARK: - General
+    
+    /**
+    *  @details Represents the Blog Name.
+    */
+    @NSManaged var name : String?
+    
+    /**
+     *  @details Stores the Blog's Tagline setting.
+     */
+    @NSManaged var tagline : String?
+    
+    /**
+     *  @details Stores the Blog's Privacy Preferences Settings
+     */
+    @NSManaged var privacy : NSNumber?
+
+    
+    
+    // MARK: - Writing
+    /**
+    *  @details Contains the Default Category ID. Used when creating new posts.
+    */
+    
+    @NSManaged var defaultCategoryID : NSNumber?
+    
+    /**
+    *  @details Contains the Default Post Format. Used when creating new posts.
+    */
+    @NSManaged var defaultPostFormat : String?
+    
+    
+    
+    // MARK: - Discussion
+    
+    /**
+    *  @details Represents whether comments are allowed, or not.
+    */
+    @NSManaged var commentsAllowed : NSNumber?
+    
+    /**
+     *  @details Contains a list of words, space separated, that would cause a comment to be automatically
+     *           blacklisted.
+     */
+    @NSManaged var commentsBlacklistKeys : String?
+    
+    /**
+     *  @details If true, comments will be automatically closed after the number of days, specified
+     *           by `commentsCloseAutomaticallyAfterDays`.
+     */
+    @NSManaged var commentsCloseAutomatically : NSNumber?
```

Comment:
I believe you can define these as `Bool` and core data will do its magic, although I haven't tried it with Swift yet